universe/Universe.cpp

```diff
@@ -2947,6 +2947,10 @@ void Universe::UpdateStatRecords() {
     }
 }
 
+void Universe::SetGameUID(const std::string& game_uid) {
+    m_game_uid = game_uid;
```

Comment:
Prefer `{ ... }` on one line for single-line functions.

Reply:
Fixed.